.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -8,8 +8,112 @@ on:
         required: true
 
 jobs:
-  build:
+  check-release:
+    strategy:
+      fail-fast: true
     runs-on: ubuntu-latest
     steps:
-      - name: Dummy step
-        run: echo "Nothing to do"
+      - uses: actions/checkout@v2
```

Comment:
Should we check the Version numbers here again

Reply:
Yes, I think it's better to check again.